app/code/Magento/OfflinePayments/view/frontend/web/js/view/payment/method-renderer/purchaseorder-method.js

```diff
@@ -25,6 +25,19 @@ define([
             return this;
         },
 
+        /**
+         *
+         * @param data
+         * @param event
+         */
+        placeOrderKeyUpdate: function (data, event) {
+            var self = this;
+            var keyCode = event.keyCode || event.which;
+            if (keyCode == '13'){
```

Comment:
Please use the mapper to get event code `magento2ce/app/code/Magento/Ui/view/base/web/js/lib/key-codes.js`